src/index.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable indent */
+import './css/index.css';
+
+const todoObjectArray = [
+  {
+    id: 1,
+    title: 'Zoom meeting at 11 am',
+    completed: false,
+  },
+  {
+    id: 2,
+    title: 'Go to church...',
+    completed: false,
+  },
+  {
+    id: 3,
+    title: 'walking during the night',
+    completed: false,
+  },
+  {
+    id: 4,
+    title: 'write some codes',
+    completed: false,
+  },
+];
+
+const taskList = document.getElementById('task-list');
+
+function renderBook() {
+  taskList.innerHTML = '';
+  todoObjectArray.forEach((todoObject, index) => {
+    taskList.innerHTML += `<ul class="todoTask" id="${index + 1}">
+                  <li><input type="checkbox" id="checkbox" onchange="ChnageCompleted(${
+                    todoObject.id
+                  })" name="checkbox"></li>        
+                  <li class="todoName">${todoObject.title}</li>    
+                  <li><img src="./images/dots.png" alt="icon"></li>   
```

Comment:
- [ ] There seems to be an issue with your images. Kindly fix that.

  ![omarabdo](https://user-images.githubusercontent.com/36542986/176704380-76948bbb-f45e-4f11-88f8-72a5c27b97e9.jpg)